components/avatar/avatar.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
You can add more meta descriptions to personalise it more. Like
<meta name="author" content="Your name" />
    <meta name="description" content="About your ui" />

Reply:
hey i have made the suggested changes you can look it up